packages/safe-ds-eda/src/components/ColumnCounts.svelte

```diff
@@ -0,0 +1,146 @@
+<script lang="ts">
+    import { addInternalToHistory } from '../apis/historyApi';
+    import { disableNonContextMenuEffects, restoreNonContextMenuEffects } from '../toggleNonContextMenuEffects';
+    import { preventClicks, table } from '../webviewState';
+
+    export let flexAsRow = true;
+
+    let contextMenuVisible = false;
+    let menuRef: HTMLElement;
+
+    const toggleMenu = function () {
+        if ($preventClicks) return;
+
+        contextMenuVisible = !contextMenuVisible;
+
+        if (contextMenuVisible) {
+            preventClicks.set(true);
+            disableNonContextMenuEffects();
+            document.addEventListener('click', handleClickOutside);
+        } else {
+            document.removeEventListener('click', handleClickOutside);
+        }
+    };
+
+    const handleClickOutside = function (event: MouseEvent) {
+        if (contextMenuVisible) {
+            if (menuRef && !menuRef.contains(event.target as Node)) {
+                preventClicks.set(false);
+                restoreNonContextMenuEffects();
+                contextMenuVisible = false;
+                document.removeEventListener('click', handleClickOutside);
+            }
+        }
+    };
+
+    const selectOption = function (callback: () => void) {
+        preventClicks.set(false);
+        restoreNonContextMenuEffects();
+        contextMenuVisible = false;
+        document.removeEventListener('click', handleClickOutside);
+        callback();
+    };
+
+    const showColumn = function (columnName: string) {
+        table.update(($table) => {
+            return {
+                ...$table!,
+                columns: $table!.columns.map((column) => {
+                    if (columnName === column.name) {
+                        return {
+                            ...column,
+                            hidden: !column.hidden,
```

Comment:
```suggestion
                            hidden: false,
```

Since the function is called "showColumn" rather than "toggleColumnVisibility", I find it clearer to be explicit here.